app.py

```diff
@@ -1,13 +1,26 @@
 from flask import Flask, render_template, request, redirect, url_for
 import os
-from trelloapp import Trello
+# from trelloapp import Trello
 import json
-from trello_class import TrelloItem
+from todo_item import TodoItem #replaced Trello
 from view_model import ViewModel
+import pymongo
+from bson import ObjectId, json_util
+from datetime import datetime
+from dotenv import load_dotenv  #to invoke .env file
+import os
 
 def create_app():
     app = Flask(__name__)
-#    app.config.from_object('flask_config.Config')  #can remove to delete secret key config
+    # app.config.from_object('flask_config.Config')  #removed secret key config
+
+    load_dotenv()
+    MONGO_LOGIN = os.getenv("MONGO_LOGIN")  # take .env from dotenv
+    MONGO_PASS = os.getenv("MONGO_PASS")  # take .env from dotenv
+
+    myclient = pymongo.MongoClient('mongodb+srv://%s:%s@cluster0.pc757.mongodb.net/ToDo?retryWrites=true&w=majority' % (MONGO_LOGIN, MONGO_PASS))    
```

Comment:
We should extract the cluster host address here (i.e. `cluster0.pc757.mongodb.net`) so that it can be specified in different environments